app/models/site_config.rb

```diff
@@ -25,6 +25,9 @@ class SiteConfig < RailsSettings::Base
   # Authentication
   field :authentication_providers, type: :array, default: %w[twitter github]
 
+  # Broadcast
+  field :welcome_notifications_live_date, type: :date, default: "2030-01-01 00:00:00 UTC".to_date
```

Comment:
@vaidehijoshi I set this to this obscure date so this will require explicit chang to enable this feature.

Reply:
I wonder if we should even have a default at all? Can we just set it to be empty? Or is that not possible?

Reply:
Yep that's possible. I went ahead and removed the default.